lighthouse-core/report/html/renderer/report-renderer.js

```diff
@@ -35,6 +35,11 @@ class ReportRenderer {
   renderReport(report, container) {
     // If any mutations happen to the report within the renderers, we want the original object untouched
     const clone = /** @type {LH.ReportResult} */ (JSON.parse(JSON.stringify(report)));
+    // Mutate the UIStrings if necessary (while saving originals)
+    ReportRenderer.stashUIStrings();
```

Comment:
if it's stashing just for this function, can it stored in a local variable instead of stashed in a static property?

Reply:
done